packages/wdio-config/src/constants.ts

```diff
@@ -1,3 +1,28 @@
+import { DesiredCapabilities, W3CCapabilities } from 'webdriver'
+import { Config, HookFunctions, MultiRemoteCapabilities } from 'webdriverio'
+import '@wdio/cucumber-framework' // extends HookFunctions with CucumberHookFunctions
```

Comment:
Instead of importing it, let's add `@wdio/cucumber-framework` to the `types` list in the sub package tsconfig and us it through the namespace. This approach will fail as we check if all imports are defined in the package.json of the sub package.